roles/openshift_logging_mux/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 ### General logging settings
-openshift_logging_mux_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default('docker.io/openshift/origin-') }}"
-openshift_logging_mux_image_version: "{{ openshift_hosted_logging_deployer_version | default('latest') }}"
+openshift_logging_mux_image_prefix: "{{ openshift_logging_image_prefix | default(__openshift_logging_image_prefix) }}"
```

Comment:
I suppose we could run into a situation where there is some sort of bug in the fluentd image that only affects mux, so you may want to run with a slightly different version of the fluentd image in mux than the one used for fluentd.  If we don't feel that is important, then I'm fine with getting rid of the separate mux prefix/version parameters.